identifiers/country-be/provinces.csv

```diff
@@ -0,0 +1,12 @@
+id,name
+ocd-division/country:be/province:an,Antwerp
+ocd-division/country:be/province:be,Brussels-Capital
```

Comment:
Brussels Capital is not a province according to https://en.wikipedia.org/wiki/ISO_3166-2:BE